cmp/grid/impl/ColumnHeader.js

```diff
@@ -34,14 +34,13 @@ export const ColumnHeader = hoistCmp({
         const sortIcon = () => {
             const activeGridSorter = impl.activeGridSorter;
             if (!activeGridSorter) return null;
+            const {abs, sort} = activeGridSorter;
 
             let icon;
```

Comment:
This seems like generally a good fix to appropriately support abs ascending, if it was ever desired by an application.